bower.json

```diff
@@ -59,12 +55,10 @@
   "resolutions": {
     "d3": "~3.5.0",
     "jquery": "~2.2.4",
-    "angular": "~1.6.6",
-    "angular-sanitize": "~1.6.6",
-    "angular-animate": "~1.6.6",
     "bootstrap-switch": "~3.3.4",
     "moment": ">=2.10.5",
     "bootstrap-select": "~1.12.2",
-    "font-awesome-sass": "~4.7.0"
+    "font-awesome-sass": "~4.7.0",
+    "angular": "1.6.9"
```

Comment:
This line is autogenerated when running `bower install -F`, keeping for now, otherwise everybody will see a local change.